test/pylib/rest_client.py

```diff
@@ -329,10 +329,10 @@ async def keyspace_flush(self, node_ip: str, keyspace: str, table: Optional[str]
 
     async def keyspace_compaction(self, node_ip: str, keyspace: str, table: Optional[str] = None) -> None:
         """Compact the specified or all tables in the keyspace"""
-        url = f"/storage_service/keyspace_compaction/{keyspace}"
```

Comment:
I'd keep the `url =` assignment and usage below
to reduce the code churn

Reply:
Ack. Done.